src/views/v2/rsvp/messages/must-login.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Block: RSVP
+ * Messages Must Login
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/messages/must-login.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @param WP_Post|int $post_id The post object or ID.
+ * @var bool $must_login Whether the user has to login to RSVP or not.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+if ( ! $must_login ) {
+	return;
+}
+
+?>
+<div class="tribe-tickets__rsvp-message tribe-tickets__rsvp-message--must-login tribe-common-b3">
+	<em class="tribe-common-svgicon tribe-tickets__rsvp-message--must-login-icon"></em>
```

Comment:
use the svg variables directly instead, we plan to remove the `svgicon` classes and use svg templates for TEC.